crates/bevy_asset/src/asset_server.rs

```diff
@@ -18,7 +18,7 @@ use thiserror::Error;
 pub enum AssetServerError {
     #[error("asset folder path is not a directory")]
     AssetFolderNotADirectory(String),
-    #[error("no AssetLoader found for the given extension")]
+    #[error("no AssetLoader found{}", format_missing_asset_ext(.0))]
     MissingAssetLoader(Option<String>),
```

Comment:
Maybe this should be:

```rust
MissingAssetLoader {
    extensions: Vec<String>
}
```

That makes the error more structured and (potentially) recoverable.